### PR TITLE
Measurement Forms

### PR DESCRIPTION
**Pull request by austingulati**

### CODE REVIEW

File: app/views/babies/_baby.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="baby span4">
+  <h3><%= link_to baby.name, baby %></h3>
+  Born <%=l baby.birth %>
```
Comment (dmastylo) [inline app/views/babies/_baby.html.erb:3]: <%=l ??

Comment (austingulati) [inline app/views/babies/_baby.html.erb:3]: Localization

Comment (dmastylo) [inline app/views/babies/_baby.html.erb:3]: KK

**Comment by dmastylo** [comment]
This ready to be merged in?

**Pull request merged**